AmplifyPlugins/Auth/AWSAuthPlugin/AWSAuthPlugin+ClientBehavior.swift

```diff
@@ -31,18 +31,27 @@ extension AWSAuthPlugin {
         fatalError()
     }
 
-    public func signIn(username: String,
-                       password: String,
+    public func signIn(username: String?,
+                       password: String?,
                        options: AuthSignInOperation.Request.Options?,
                        listener: AuthSignInOperation.EventListener?) -> AuthSignInOperation {
-        fatalError()
+        let options = options ?? AuthSignInRequest.Options()
+        let request = AuthSignInRequest(username: username,
+                                        password: password,
+                                        options: options)
+        let signInOperation = AWSAuthSignInOperation(request,
+                                                     authenticationProvider: authenticationProvider,
+                                                     listener: listener)
+        queue.addOperation(signInOperation)
+        return signInOperation
     }
 
     public func signInWithSocial(provider: AuthSocialProvider,
```

Comment:
I know we haven't done an API naming yet, but this would be Swiftier as `signIn(withSocialProvider provider: AuthSocialProvider...`

Reply:
Yes, will update after the naming.